tasks/setup-RedHat.yml

```diff
@@ -19,19 +19,20 @@
 - name: Download specific Jenkins version.
   get_url:
     url: "{{ jenkins_pkg_url }}/jenkins-{{ jenkins_version }}-1.1.noarch.rpm"
-    dest: "/tmp/jenkins.rpm"
+    dest: "/tmp/jenkins_{{ jenkins_version }}.rpm"
   when: jenkins_version is defined
 
 - name: Check if we downloaded a specific version of Jenkins.
   stat:
-    path: "/tmp/jenkins.rpm"
+    path: "/tmp/jenkins_{{ jenkins_version }}.rpm"
+  when: jenkins_version is defined
```

Comment:
I wonder if this `when` is causing the fail (if CentOS runs without a specific version defined)...